Add anijs to the list

josephrexme opened the pull request:
Anijs uses data attributes for animations with no need for code initialization

josephrexme committed:
Add anijs to the list

LeaVerou commented:
Sweet, thanks! Nice project, it's like ifttt for animations :)
If you could fix the indentation, that'd be appreciated.

josephrexme commented:
Thank you I fixed the indentation on #53 . I'm sorry it looked normal to me on my 2 spaces configured editor till I switched to tabbed view and saw it
